contracts/.prettierignore

```diff
@@ -22,6 +22,7 @@ LinkToken.json
 typechain
 **/vendor
 src/v0.8/ccip/**
+src/v0.8/workflow/**
```

Comment:
any reason we ignore prettier,  i used to rely on prettier to format for me :)

Reply:
:D we're using forge fmt right now in our repo as it's a bit faster :D